requirements_py3.txt

```diff
@@ -1,6 +1,10 @@
-nose>=1.3.1
-yaswfp>=0.4
-beautifulsoup4==4.6.0
-m3u8==0.3.2
-slimit==0.8.1
-ply==3.4
```

Comment:
Don't remove these dependencies! They are needed to run the tests. Try `./test` (which needs also to be updated, see that runs flake8 using python2).

Reply:
I've updated the dependecies and the test file for running ./test